core/app/models/spree/export.rb

```
@@ -188,7 +188,11 @@ def type_for_model(model)
 
       # eg. Spree::Exports::Products => Spree::Product
       def model_class
-        "Spree::#{to_s.demodulize.singularize}".constantize
+        klass = "Spree::#{to_s.demodulize.singularize}".safe_constantize
+
+        raise NameError, "Missing model class for #{to_s}" unless klass
```

RuboCop (1.81.1)

[warning] Use `self` instead of `Object#to_s` in interpolation.

(Lint/RedundantStringCoercion)